bin/salt_python_wrapper

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -u
+
+for py in 'python3' 'python'; do
+  exe=\$(type -p \${py})
+  if [ -n "\${exe}" ]; then
+    if \${exe} -c 'import salt.config'; then
+      \${exe} "\$@"
+      exit \$?
+    fi
+  fi
+done
+
+echo "No usable python version found, check if python-salt-library is installed    !" 1>&2
```

Comment:
If you get this error message, you would try to find a package with the name "python-salt-library". I don't think such a package exists. Can you revise the message please.

Reply:
Changed to "check if python or python3 can import salt.config".